azure/function-app/communication-service/README.md

```diff
@@ -1,5 +1,46 @@
 # communication-service
 
-This example provisions a Function App that can connect to an Azure Communication Service bundle. This example is using the Azure Email Communication Services resource, which is still in preview. Provision an Email service resource in the Azure portal [here](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Communication%2FEmailServices).
+This example provisions a Function App that can connect to an Azure Communication Service bundle. This example is using the Azure Email Communication Services resource, which is still in public preview. 
 
-Once the Email service is generally available, we will build the bundle to deploy this from within Massdriver.
\ No newline at end of file
+Once the Email service is generally available, we will build the bundle to deploy this from within Massdriver.
+
+## Use case
+This application example will let you send an email to any recipient(s) using API requests.
+
+## Deployment guide
+Below are the steps needed to deploy this example in Massdriver:
+1. Clone the repo
+2. `cd` into `application-examples/azure/function-app/communication-service` and run `mass app publish`. If you don't have your Massdriver API key or CLI, follow these [steps](https://docs.massdriver.cloud/bundles/walk-through#generate-massdriver-api-token)
```

Comment:
These are really good steps! We should talk about maybe adding a `preview.json` that sets up an entire env? There's no pattern there ATM, just speaking out loud.

Reply:
Anything that makes it easier for the customer to go through these steps, haha

Reply:
Yeah these are veerryyy valuable now. Knowing us, we can do better with time. You're paving the way to the best experience ATM. 💪

Reply:
Definitely interested in a `preview.json` whenever you have time